tools/editor/scenes_dock.cpp

```diff
@@ -959,6 +960,17 @@ void ScenesDock::_file_option(int p_option) {
 
 
 			String path = files->get_item_metadata(idx);
+			if (p_option == FILE_SHOW_IN_EXPLORER) {
+			  String dir = DirAccess::get_full_path("res://", DirAccess::ACCESS_FILESYSTEM);
+				const int res_begin = String("res://").length();
+				const int last_sep = path.find_last("/");
+				if (last_sep > res_begin) {
+					dir += "/";
+					dir += path.substr(res_begin, last_sep - res_begin);
+				}
+				OS::get_singleton()->shell_open(String("file://")+dir);
```

Comment:
`get_current_dir()` return unfixed `current_dir` as [DirAccess::_get_root_path()](https://github.com/godotengine/godot/blob/a895e2e372c9df2a9972f38134ae0ea9c8bf45a8/core/os/dir_access.cpp#L36-L46) returns `""` for `ACCESS_FILESYSTEM`. 

So I think this problem also exists in windows and osx.